assets/js/components/ErrorNotice.test.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * ErrorNotice component tests.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry, provideModules } from '../../../tests/js/utils';
+import {
+	ERROR_CODE_MISSING_REQUIRED_SCOPE,
+	ERROR_REASON_INSUFFICIENT_PERMISSIONS,
+} from '../util/errors';
+import { fireEvent, render } from '../../../tests/js/test-utils';
+import ErrorNotice from './ErrorNotice';
+import { MODULES_ANALYTICS } from '../modules/analytics/datastore/constants';
+
+describe( 'ErrorNotice', () => {
+	let registry;
+	let invalidateResolutionSpy;
+	const moduleName = 'test-module';
+
+	beforeEach( () => {
+		registry = createTestRegistry();
+		provideModules( registry, [
+			{ slug: moduleName, name: 'Test Module' },
+		] );
+		invalidateResolutionSpy = jest.spyOn(
+			registry.dispatch( MODULES_ANALYTICS ),
+			'invalidateResolution'
+		);
+	} );
+
+	afterEach( () => invalidateResolutionSpy.mockReset() );
```

Comment:
We aren't looking for the return value here, so let's not use an implicit return here.

```suggestion
	afterEach( () => {
		invalidateResolutionSpy.mockReset();
	} );
```